### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by Valentyn-Radobenko**
- [DEMO LINK](https://Valentyn-Radobenko.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Valentyn-Radobenko.github.io/layout_product-cards/report/html_report/)
- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/styles/card.scss [hunk 1]
```diff
@@ -0,0 +1,106 @@
+@import 'variables';
+
+.card {
+  width: $card-width;
+  height: $card-height;
```
Comment (Moroz-Dmytro) [inline src/styles/card.scss:5]: Better not to fix height

File: src/styles/card.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  border: $card-border;
+  border-radius: $card-border-radius;
+  box-shadow: $card-shadow;
+  font-family: $font-family;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: $card-padding;
+  background-color: #fff;
+  transition: box-shadow 0.3s;
```
Comment (Moroz-Dmytro) [inline src/styles/card.scss:15]: The order of your styles is important. Here is useful link - https://9elements.com/css-rule-order/

File: src/styles/card.scss [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  height: $card-height;
+  border: $card-border;
+  border-radius: $card-border-radius;
+  box-shadow: $card-shadow;
+  font-family: $font-family;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: $card-padding;
+  background-color: #fff;
```
Comment (Moroz-Dmytro) [inline src/styles/card.scss:14]: Create variable for all colors

File: src/styles/card.scss [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  }
+
+  &__title {
+    display: block;
+    width: 166px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+    margin-bottom: 4px;
```
Comment (Moroz-Dmytro) [inline src/styles/card.scss:34]: Use only `margin-top` or `margin-bottom`

File: src/styles/card.scss [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    margin-left: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 166px;
+    font-size: 10px;
+    font-weight: 400;
```
Comment (Moroz-Dmytro) [inline src/styles/card.scss:55]: Add default `font-family` and `font-size` to your `html`, not to repeat it in every class

File: src/styles/card.scss [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    &-value {
+      font-size: 16px;
+      font-weight: $price-font-weight;
+      color: $price-color;
+    }
+  }
+
+  &__link {
+    display: block;
+    width: 166px;
```
Comment (Moroz-Dmytro) [inline src/styles/card.scss:88]: Do you really need to fix width here?

File: src/styles/card.scss [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    color: $card-reviews-color;
+    margin-bottom: 24px;
+
+    &-text {
+      display: block;
+    }
+  }
+
+  &__price {
+    width: 166px;
```
Comment (Moroz-Dmytro) [inline src/styles/card.scss:66]: Do you really need to fix width here?

File: src/styles/card.scss [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+    margin-bottom: 16px;
+    align-self: flex-start;
+    margin-left: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 166px;
```
Comment (Moroz-Dmytro) [inline src/styles/card.scss:53]: Do you really need to fix width here?

File: src/styles/variables.scss [hunk 9]
```diff
@@ -0,0 +1,22 @@
+$card-width: 198px; // ширина карти з урахуванням бордюру буде 200px
```
Comment (Moroz-Dmytro) [inline src/styles/variables.scss:1]:
```suggestion
$card-width: 200px;
```

And add `box-sizing: border-box;`

File: src/styles/variables.scss [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+$card-height: 408px;
+$card-border: 1px solid #fff;
+$card-border-radius: 5px;
+$card-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+$font-family: Roboto, sans-serif;
```
Comment (Moroz-Dmytro) [inline src/styles/variables.scss:6]: Don't need variable for `font-family`

**Review by Moroz-Dmytro — CHANGES_REQUESTED** [review]
Use more semantic tags, like `h2`, `ul`, `p`...

File: src/index.html [hunk 11]
```diff
@@ -9,10 +9,50 @@
...
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="iMac"
```
Comment (BudnikOleksii) [inline src/index.html:22]: alt attribute should describe the image content
image

**Review by BudnikOleksii — APPROVED** [review]
Good work 👍

**Pull request closed without merging**